src/streamlink/plugins/arconai.py

```diff
@@ -1,34 +1,35 @@
 import re
 
 from streamlink.plugin import Plugin
-from streamlink.plugin.api import http, validate
-from streamlink.plugin.api.utils import parse_json
+from streamlink.plugin.api import http
+from streamlink.plugin.api import useragents
 from streamlink.stream import HLSStream
 
-_url_re = re.compile(r"https://www.arconaitv.me/([^/]+)/")
+_url_re = re.compile(r'''https?://(www\.)?arconaitv\.me/stream\.php\?id=\d+''')
+_playlist_re = re.compile(r'''source\ssrc=["'](?P<url>[^"']+)["']''')
 
-SOURCES_RE = re.compile(r" data-item='([^']+)' ")
-USER_AGENT = "Mozilla/5.0 (Windows NT 6.1) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/58.0.3029.81 Safari/537.36"
 
 class ArconaiTv(Plugin):
     @classmethod
     def can_handle_url(cls, url):
         return _url_re.match(url)
 
     def _get_streams(self):
-        page = http.get(self.url)
-        match = SOURCES_RE.search(page.text)
-        if match is None:
-            return
+        headers = {
+            'User-Agent': useragents.CHROME,
+            'Referer': self.url
+        }
 
-        sources = parse_json(match.group(1))
-        if "sources" not in sources or not isinstance(sources["sources"], list):
+        res = http.get(self.url, headers=headers)
+
+        match = _playlist_re.search(res.text)
+        if match is None:
```

Comment:
well this might work here,

but since i use a template for plugins,
it would make more work for me if i had to do this always,
when there is more than just `url` from the regex

it's easier to copy and paste with less changes 🦅 

```py
'''
source: -

Plugin for
- https://example.com
'''
import re

from streamlink.plugin import Plugin
from streamlink.plugin.api import http
from streamlink.stream import HLSStream

_url_re = re.compile(r'''https?://(?:\w+\.)?example\.com''')
_playlist_re = re.compile(r'''{\s+file:\s["'](?P<url>[^"']+)["']''')


class PLUGINNAME(Plugin):
    @classmethod
    def can_handle_url(cls, url):
        return _url_re.match(url)

    def _get_streams(self):
        res = http.get(self.url)

        match = _playlist_re.search(res.text)
        if match is None:
            return

        url = match.group('url')
        if url.startswith('//'):
            url = 'https:{0}'.format(url)

        if url:
            self.logger.debug('HLS URL: {0}'.format(url))
            yield 'live', HLSStream(self.session, url)

__plugin__ = PLUGINNAME

```